### PR TITLE
add translation for get-started-calico

### PR DESCRIPTION
**Pull request by Jeanine-tw**
---
name: Pull request
about: Tell us about your contribution
labels: ["pr/release/none-required"]
---

---
## Thanks for contributing !

Before submitting a pull request, make sure you read about our Contribution notice here: <https://spidernet-io.github.io/spiderpool/contributing/development/pullrequest/#need-review>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

**Special notes for your reviewer**:

**make sure your commit is signed off**

### CODE REVIEW

**Comment by weizhoublue** [comment]
解决 ci 中报错的 格式问题

File: docs/usage/get-started-calico.md [hunk 1]
```diff
@@ -0,0 +1,283 @@
+# Calico Quick Start
+
+**English** | [**简体中文**](./get-started-calico-zh_CN.md)
+
+SpiderPool is able to provide static IPs to deployment, StatefulSet, and other types of applications in underlay networks. In this page, we'll introduce how to build a complete Underlay network environment in Calico + BGP mode, and integrate it with SpiderPool to enable fixed IP addresses for applications. This solution meets the following requirements:
```
Comment (windsonsea) [inline docs/usage/get-started-calico.md:5]:
```suggestion
SpiderPool is able to provide static IPs to deployments, StatefulSets, and other applications in underlay networks. In this page, we'll introduce how to build a complete underlay network environment in Calico + BGP mode, and integrate it with SpiderPool to enable fixed IP addresses for applications. This solution meets the following requirements:
```
哪种写法是对的？
- Spiderpool
- spiderpool
- SpiderPool

File: docs/usage/get-started-calico.md [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+
+* Assign static IP addresses to applications
+
+* IP pools dynamically scale based on replica counts
```
Comment (windsonsea) [inline docs/usage/get-started-calico.md:9]:
```suggestion
* Scale IP pools dynamically based on replica counts
```
这个词也是，哪个是对的？
- IP pool
- IPPool
- IPpool

File: docs/usage/get-started-calico.md [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+
+* External clients outside the cluster can access applications without their IPs
```
Comment (windsonsea) [inline docs/usage/get-started-calico.md:11]:
```suggestion
* Enable external clients outside a cluster to access applications without their IPs
```
3 个句式需要一致：v + n

**Review by windsonsea — COMMENTED** [review]
可以创建一个 glossary 页面，把这些 term 的写法、含义固定下来：
- Spiderpool
- IPPool
- SpiderNet
- ...

**Comment by ty-dc** [comment]
文档中的 markdown 的缩进还有一点问题
image

File: docs/usage/get-started-calico.md (hunk 3 above)
Comment (Jeanine-tw) [reply docs/usage/get-started-calico.md]: 已改

File: docs/usage/get-started-calico.md (hunk 1 above)
Comment (Jeanine-tw) [reply docs/usage/get-started-calico.md]: 已统一：Spiderpool

File: docs/usage/get-started-calico.md (hunk 2 above)
Comment (Jeanine-tw) [reply docs/usage/get-started-calico.md]: IP pool

**Comment by Jeanine-tw** [comment]
> 可以创建一个 glossary 页面，把这些 term 的写法、含义固定下来：
> 
> * Spiderpool
> * IPPool
> * SpiderNet
> * ...
> 
> 一致性 consistency 很重要：term、句式、文档和逻辑结构、插图

已经修改了。这次时间有点紧，后面建一个风格指南

**Pull request merged**